metadata_xml/template_functions.py

```diff
@@ -33,25 +42,31 @@ def normalize_datestring(datestring, format_='default'):
 
 
 def list_all_languages_in_record(record):
-    '''So that we can list all languages used in the otherLocale section of XML'''
-    def list_keys_in_dict(d, all_keys=[]):
-        # recursive function, works on nested dict for example
-        for k, v in d.items():
-            if isinstance(v, dict):
-                all_keys.append(k)
-                list_keys_in_dict(v, all_keys)
+    '''Lists all languages used, so that we can list all
+    languages used in the otherLocale section of XML'''
+    def list_keys_in_record(record_subset, all_keys=[]):
+        'recursive function, works on nested dict for example'
```

Comment:
Minor but docstrings should usually be triple double quotes per PEP:
https://www.python.org/dev/peps/pep-0257/

Reply:
fixed thx